packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -116,6 +117,8 @@ const IconSwitcher = ({ iconName, ...iconProps }: IconSwitcherProps) => {
       return <GearIcon {...iconProps} />;
     case 'ClockWithArrowIcon':
       return <ClockWithArrowIcon {...iconProps} />;
```

Comment:
I suppose this icon can be removed since it is being replaced by `ListWithSearch`

Reply:
True, I suppose there is no need to keep it around as an option for future devs.